helix-core/src/match_brackets.rs

```diff
@@ -15,11 +15,13 @@ const PAIRS: &[(char, char)] = &[
 
 // Returns the position of the matching bracket under cursor.
 //
-// If the cursor is one the opening bracket, the position of
-// the closing bracket is returned. If the cursor in the closing
+// If the cursor is on the opening bracket, the position of
+// the closing bracket is returned. If the cursor on the closing
 // bracket, the position of the opening bracket is returned.
 //
 // If the cursor is not on a bracket, `None` is returned.
+//
+// If no matching bracket is found, `None` is returned.
```

Comment:
This should probably be a doc-comment (`///`) as opposed to a regular comment

Reply:
https://github.com/helix-editor/helix/pull/4288/commits/f5cb382af064d6db6df2e5da89913ae24e363ee5